switchsdk/src/main/java/net/gini/switchsdk/TakePictureContract.java

```diff
@@ -43,6 +43,13 @@
 
         void exitSdk(final int resultCode);
 
+        /**
+         * Exits the SDK without showing the "your document has been analyzed" screen.
+         *
+         * @param resultCode the activity's result code
+         */
+        void exitSdkWithoutAnalyzedScreen(final int resultCode);
```

Comment:
Why not just use the regular `exitSdk ` method?
Is there a case where users will have different things to do in `exitSdk` and `exitSdkWithoutAnalyzedScreen`. My guess would be that they never support both.

Reply:
In case the "Finish" button is pressed, the "Analyzing Completed" screen should never be shown. In case the transition is due to the pro-active analyzing, the "Analyzing Completed" screen should be shown depending on what the SDK creator wanted.